### PR TITLE
create passing

### PR DESCRIPTION
**Pull request by lkatalin**
```
running 3 tests
test secs::test::align ... ok
test secs::test::size ... ok
test test::enclave_create ... ok

```

### CODE REVIEW

File: sgx-sys/src/lib.rs [hunk 1]
```diff
@@ -58,6 +58,8 @@ mod test {
         secs.base = 8192;
         secs.ssa_frame_size = 4096;
         secs.attributes = secs::Attributes::MODE_64_BIT;
+        secs.miscselect = secs::MiscSelect::EXINFO;
+        secs.xfrm = 0x3;
```
Comment (npmccallum) [inline sgx-sys/src/lib.rs]: First off, awesome job!

The big question is: what belongs in `xfrm`?

Currently, `xfrm` is defined as a `u64` (https://github.com/enarx/enarx/blob/master/sgx-sys/src/secs.rs#L50). This probably isn't correct. Is this a bundle of flags? If so, we should turn this into a bitflags type (see: https://github.com/enarx/enarx/blob/master/sgx-sys/src/secs.rs#L28).

Comment (lkatalin) [reply sgx-sys/src/lib.rs]: Thanks - I'm looking into this today.

Comment (npmccallum) [reply sgx-sys/src/lib.rs]: @lkatalin I can confirm that your patch indeed does fix the tests.

Comment (lkatalin) [reply sgx-sys/src/lib.rs]: @npmccallum That's great! 

I turned the `xfrm` into a set of bigflags based on the description of `xcr0` from https://en.wikipedia.org/wiki/Control_register and turned on the last two bits, so the test still passes. I'm still not sure if this is the correct value. I may send an email about it.

Comment (npmccallum) [reply sgx-sys/src/lib.rs]: Good enough for me.

**Comment by lkatalin** [comment]
@npmccallum I updated the commit message; hopefully this is better?

**Pull request merged**